android/app/src/main/kotlin/com/jabook/app/jabook/compose/feature/player/PlaybackSpeedSheet.kt

```
@@ -67,6 +87,17 @@ public fun PlaybackSpeedSheet(
 ) {
     // Local state for slider - allows real-time preview
     var sliderSpeed by remember { mutableFloatStateOf(currentSpeed) }
+    val hapticFeedback = LocalHapticFeedback.current
+    val recentSpeeds =
+        rememberSaveable(
+            saver =
+                listSaver(
+                    save = { it.toList() },
+                    restore = { it.toTypedArray().let { array -> mutableStateListOf(*array) } },
```

detekt (1.23.8)

[warning] In most cases using a spread operator causes a full copy of the array to be created before calling a method. This may result in a performance penalty.

(detekt.performance.SpreadOperator)